packages/ckeditor5-package-generator/lib/templates/common/ckeditor5-metadata.json

```diff
@@ -1,10 +1,10 @@
 {
 	"plugins": [
 		{
-			"name": "My plugin",
-			"className": "MyPlugin",
+			"name": "<%= packageName.spacedOut %>",
+			"className": "<%= packageName.pascalCase %>",
 			"description": "Adds text to the editor.",
-			"path": "src/myplugin.<%= programmingLanguage %>",
+			"path": "src/<%= packageName.lowerCase %>.<%= programmingLanguage %>",
 			"uiComponents": [
 				{
 					"name": "myButton",
```

Comment:
Shouldn't we rename the UI component too? For instance, when creating the `ckeditor5-led-desktop` package, the button would be called `ledDesktopButton`.